packages/just-scripts/package.json

```diff
@@ -14,6 +14,7 @@
   "license": "MIT",
   "main": "./lib/index.js",
   "dependencies": {
+    "@microsoft/api-extractor": "^7.0.34",
```

Comment:
we don't add deps here as a principle - deps are controlled on the side of the user. I'm thinking of calling take api-extractor as CLI rather than using the API in the future...

Reply:
What would be the best way to ensure type safety in apiExtractorTask instead?